dependencies.yaml

```diff
@@ -67,13 +67,15 @@ dependencies:
       - output_types: conda
         packages:
           - c-compiler
-          - cmake>=3.23.1,!=3.25.0
+          - cmake>=3.26.4
```

Comment:
I updated this cmake pinning to match the rest of RAPIDS -- and the pinning in meta.yaml.